### ISSUE TITLE
Feature Request: Make the Plunker Editor mobile-friendly

### ISSUE THREAD

**Issue by ramtob** [filearts/plunker#110]
Hello,

In these days it is becoming more and more desirable to have sites accessible to mobile browsers. Even elaborate web apps such as Plunker. While Plunker is otherwise an awesome service, it fails completely on mobile browsers.

It would be ideal to have an actually responsive app, such as Codepen has recently become. But it will also be a great advance if Plunker will just display the same on mobile as on desktop browsers, with the accompanying need to zoom and scroll on mobile (see for example the Jsbin site). Unlike the current state, where Plunker displays on mobile browsers squashed, unreadable and unusable.

TIA

**Comment by hemulin** [comment]
+1

**Comment by elbuki** [comment]
+1

**Comment by Sk3p** [comment]
+1

**Comment by idanba** [comment]
+1

**Comment by ggoodman** [comment]
I agree that this would be great. What is preventing this is my own personal bandwidth for writing code.

**Comment by paraflu** [comment]
+1

**Comment by ramtob** [comment]
@ggoodman Naturally you would be busy. Do you perhaps have an idea _what_ can be done, in the code, to improve the responsiveness of Plunker?

**Comment by touficbatache** [comment]
+1

**Comment by beeisabelm** [comment]
+1

_Sent from my Samsung SM-N900T using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by kepiz** [comment]
+1

**Comment by sierraindiagolf** [comment]
+1

**Comment by vrndeva** [comment]
+1

**Comment by blacksoulgem95** [comment]
+1
